src/reducers/index.js

```diff
@@ -125,6 +130,7 @@ const handlers = {
   SET_ITEMS: setItemsOrSelections,
   CHANGE_SELECTIONS: setItemsOrSelections,
   RETRIEVE_ERROR: setRetrieveError,
+  RETRIEVE_RESOURCE_TEMPLATE_STARTED: clearRetrieveError,
```

Comment:
This could potentially cause confusion. Would prefer to have these aligned.

Reply:
@justinlittman I disagree.   The message is "what happened" and the method is "what to do now that that event occurred".   When X happens, then do Y.

Reply:
OK, then we can agree to disagree.